apps/src/gamelab/GameLab.js

```diff
@@ -729,18 +729,33 @@ function p5KeyCodeFromArrow(idBtn) {
   }
 }
 
+function domIdFromArrow(idBtn) {
+  switch (idBtn) {
+    case ArrowIds.SPACE:
+      return 'studio-space-button';
+  }
+}
```

Comment:
I'm not a fan of the implied `return undefined` in this method; can we make it explicit?